BRANCHING_RELEASE_STRATEGY.md

```diff
@@ -0,0 +1,178 @@
+- Each commits must be signed correctly
+- WIP status must be cleared
+
+### Release branches
+
+Release branches, which pattern is  `release-v*..*`, are created from the corresponding tag `v*.*` each time a corrective release is needed.
+Only [maintainers](MAINTAINERS.md) can create or force-push into these branches.
+Other members must create a pull request to do push into these branches and pass the same checks as the `main` branch.
+These branches are always up to the most recent patch of the release.
+
+### Integration branches
+
+There are two different types of integration branches:
+- Branches that are created to support a specific major feature or to adapt to a major breaking change present in a SNAPSHOT release of another PowSyBl repository; their pattern is `integration/<other_repository_name>-<SNAPSHOT_release>` or `integration/<major_feature_name>`
+- Branches preparing a new XIIDM version, which means updating of several files that can be awkward to review. Their pattern can be `evolution_xiidm/<xiidm_version>` or `xiidm_version_<xiidm_version>`
+
+Only [maintainers](MAINTAINERS.md) can create or force-push into these branches.
+Other members must create a pull request to do push into these branches and pass the same checks as the `main` branch.
+
+## Release cycle
+
+A new release train of the framework occurs roughly every two months.
+
+In case of structural issues, corrective releases can also be published for some repositories.
+These corrective releases are motivated by users' demand so don't hesitate to [contact us](https://www.powsybl.org/pages/community/).
+
+There is also always a possibility to release a module outside this agenda if necessary,
+please [contact us](https://www.powsybl.org/pages/community/) for further discussion.
+
+A few weeks before the release train, one or two release candidates of [powsybl-core](https://github.com/powsybl/powsybl-core) are
+usually published and available for testing and migration of dependent repositories.
+A release candidate should be up for at least one week before the release is published.
+On average, it is up for two weeks.
+
+Our release train consists in the release of:
+- [powsybl-core and its sub-modules](https://github.com/powsybl/powsybl-core)
+- [powsybl-open-loadflow](https://github.com/powsybl/powsybl-open-loadflow)
+- [powsybl-diagram and its sub-modules](https://github.com/powsybl/powsybl-diagram)
+- [powsybl-dynawo and its sub-modules](https://github.com/powsybl/powsybl-dynawo)
+- [pypowsybl](https://github.com/powsybl/pypowsybl)
+- [powsybl-entsoe and its sub-modules](https://github.com/powsybl/powsybl-entsoe)
+
+For each released repository:
+- a release note is written by one of the repository's committer
+- in case of breaking changes, a migration guide is written by one or several of the repository's developers
+- its latest version is updated on the [repositories' pages](../documentation/developer/repositories/index.md)
```

Comment:
the link is missing a part, should we indicate https://github.com/powsybl/powsybl.github.io/blob/main/pages/documentation/developer/repositories/index.md?

Reply:
indeed.... I copied and paster a wrong reference, we've moved pages since